### PR TITLE
feat: add lambdaCheckParameters

### PR DESCRIPTION
**Pull request by marcosalves-bitg**
This PR adds an generic lambdaCheckParameters, with custom keys to verify their existence

### CODE REVIEW

File: src/lambda/lambdaCheckParameters.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import type { LambdaErrors } from './interfaces'
+
+export const lambdaCheckParameters = (object: object, indexes: string[]): object => {
+  const errors: LambdaErrors = {}
+  if (Object.keys(object).length > 0 && indexes.length > 0) {
+    indexes.forEach(index => {
+      errors[index] = { ...(!object[index] ? { error: 'undefined' } : '') }
+      // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
+      if (index in object) delete errors[index]
```
Comment (Viserion77) [inline src/lambda/lambdaCheckParameters.ts:9]: Não teria como fazer um if só aqui?

**Review by Viserion77 — CHANGES_REQUESTED** [review]

File: src/lambda/lambdaCheckParameters.ts (hunk 1 above)
Comment (alanneves) [inline src/lambda/lambdaCheckParameters.ts:9]: Why are u doing this verification?

**Review by alanneves — CHANGES_REQUESTED** [review]

File: src/lambda/lambdaCheckParameters.ts (hunk 1 above)
Comment (marcosalves-bitg) [reply src/lambda/lambdaCheckParameters.ts:9]: Sorry, just copied it... didn't improved it...

Comment (marcosalves-bitg) [reply src/lambda/lambdaCheckParameters.ts:9]: Teria!!!

File: src/lambda/lambdaCheckParameters.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export const lambdaCheckParameters = (object: object, indexes: string[]): object => {
+  const errors = {}
```
Comment (alanneves) [inline src/lambda/lambdaCheckParameters.ts:2]: What I recommend to make to improve this function:

- Do an early return, make the if already returning `{ object: undefined }`
- And later, you can just return a index.reduce returning the errors or the empty object

@Adapcon/bomb-squad @Adapcon/coca-squad  what you guys think about that?

**Review by alanneves — CHANGES_REQUESTED** [review]

File: src/lambda/lambdaCheckParameters.ts (hunk 2 above)
Comment (marcosalves-bitg) [reply src/lambda/lambdaCheckParameters.ts]: Done!!! :D

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by alanneves — APPROVED** [review]
LGTM

**Pull request merged**